docs/AppService/AppServiceActivation/assets/AppServiceActivation_SetAsDefault_Success.puml

```diff
@@ -0,0 +1,11 @@
+@startuml
+
+participant Core
+participant HMI
+
+HMI -> Core : AppServiceActivation\n(serviceID: "providerID", \nsetAsDefault: true)
```

Comment:
According to the HMI spec, activate is a mandatory parameter so we should probably add that parameter in the request and response of this example